Libraries/NativeAnimation/RCTNativeAnimatedNodesManager.m

```diff
@@ -61,10 +76,69 @@ - (instancetype)initWithBridge:(nonnull RCTBridge *)bridge
     _animationNodes = [NSMutableDictionary new];
     _eventDrivers = [NSMutableDictionary new];
     _activeAnimations = [NSMutableSet new];
+    _uiManagerOperationQueue = [NSMutableArray new];
+    _layoutProps = [NSMutableArray arrayWithArray:@[
```

Comment:
Let's make this a `NSSet` so we can remove logic to avoid duplicates in `addEnqueuedUpdateProp`. Might also be slightly more performant (although array and set performance is very similar for lookups with a small amount of items).

Can probably also change the android implementation to use a Set too.

Reply:
Done. Android already implemented this with Set.